phf_macros/Cargo.toml

```diff
@@ -10,11 +10,15 @@ repository = "https://github.com/sfackler/rust-phf"
 [lib]
 proc-macro = true
 
+[features]
+unicase_support = ["unicase", "phf_shared/unicase"]
```

Comment:
Shouldn't we call it `unicase` for consistency?

Reply:
I am not sure a feature can have the same name as a dependency.

Reply:
Yes, it can. See the feature in `phf`.

Reply:
But there is no `unicase` dependency in `phf`.
And `unicase_support` is the original feature name: See https://github.com/sfackler/rust-phf/blob/master/phf_macros/tests/test.rs#L240 and https://github.com/sfackler/rust-phf/blob/master/phf_macros/tests/test.rs#L253

Reply:
I think it's okay to use the same name as previous 👍